crates/tessellation/src/fill.rs

```diff
@@ -192,19 +194,20 @@ impl Span {
 
 struct Spans {
     spans: Vec<Span>,
-    pool: Vec<Box<MonotoneTessellator>>,
+    pool: Vec<MonotoneTessellator>,
```

Comment:
This needs to be reverted to the Boxed version (the whole pull #621 optimization thing).